modules/features/pages/plugins.adoc

```diff
@@ -1,3 +1,125 @@
 = Plugins
 
-This is _Section 2_ of _Chapter 3_ in the *hello* quick course....
\ No newline at end of file
+**Plugins** are Modular extensions that enhance the functionality and capabilities of the platform.
+They were designed to add specific features, integrations, or customizations on top of the base RHDH platform.
+
+Plugins can provide new pages in the developer portal, new components, new APIs, new services, or even complete workflows within RHDH.
+Plugins can be of three different types:
+
+Frontend Plugins:: A static front end user interface that is written in React + Typescript that integrates with internal or external APIs to render data in the IDP
+Backend Plugins:: A backend API that is written in Typescript that exposes an API to frontend plugins
+Frontend + Backend plugins:: A mix of frontend and backend Typescript code that provides a certain feature of functionality
+
+Red Hat Developer Hub (RHDH) has a very flexible and extensive set of plugins that provide most of the functionality in the product.
+The core features like Software Catalog, Templates, TechDocs and Search are themselves built as plugins in RHDH.
+
+There are hundreds of plugins created by the Backstage community that you can use to customize your instance of RHDH, and adapt the IDP to your needs.
+
+image::rhdh-plugins.png[title=RHDH Plugins,width=800px]
+
+== Red Hat Supported Plugins
+
+Red Hat actively participates in the thriving Backstage community by developing and supporting a number of useful plugins that are pre-integrated into RHDH by default.
+Some of the most important ones are listed below. There are plans to develop more plugins that integrate with the Red Hat product ecosystem.
+
+=== Tekton Plugin
+
+The Tekton plugin enables you to visualize the results of CI/CD pipeline runs on your Kubernetes or OpenShift clusters.
+The plugin allows users to visually see high level status of all associated tasks in the pipeline for their applications.
+
+image::rhdh-tekton-plugin.png[title=RHDH Tekton Plugin,width=800px]
```

Comment:
This image (at least the rendered one in the HTML you sent out) is of the Gitlab pipelines.  It says `Gitlab pipelines: my-quarkus-app` at the top

Reply:
fixed. added correct image for tekton plugin. Nice catch! Thanks!